admin/server/api/list.js

```diff
@@ -40,8 +40,8 @@ module.exports = function (req, res) {
 				if (!field) return sendError('invalid field provided');
 
 				_.each(req.query.filters, function (value, key) {
-					query.where(key).equals(value ? value : null);
-					count.where(key).equals(value ? value : null);
+					query.where(key).equals(value);
```

Comment:
These aren't unnecessary, `null` vs `false` has an important impact on the query. Note that and include // ignore statements for these lines.